packages/schema/lib/schemas/ReadSchema.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const makeSchema = require('../utils/makeSchema');
+
+const BasicDisplaySchema = require('./BasicDisplaySchema');
+const BasicActionOperationSchema = require('./BasicActionOperationSchema');
+const KeySchema = require('./KeySchema');
+
+module.exports = makeSchema(
+  {
+    id: '/ReadSchema',
+    description: 'How will Zapier fetch resources from your application?',
+    type: 'object',
+    required: ['key', 'noun', 'display', 'operation'],
+    examples: [
+      {
+        key: 'recipes',
+        noun: 'Recipes',
+        display: {
+          label: 'Recipes',
+          description: 'A Read to let Zapier fetch recipes.'
+        },
+        operation: {
+          perform: '$func$0$f$',
+          sample: {
+            id: 1,
+            firstName: 'Walter',
+            lastName: 'Sobchak',
+            occupation: 'Bowler'
+          }
+        }
+      }
+    ],
+    antiExamples: [
+      {
+        display: {
+          label: 'Get Users',
+          description: 'Retrieve an index of users.'
+        },
+        operation: {
+          description: 'Define how this search method will work.'
+        }
+      }
+    ],
+    properties: {
+      key: {
+        description: 'A key to uniquely identify this trigger.',
+        $ref: KeySchema.id
+      },
+      noun: {
+        description:
+          'A noun for this trigger that completes the sentence "triggers on a new XXX".',
+        type: 'string',
+        minLength: 2,
+        maxLength: 255
+      },
+      display: {
+        description: 'Configures the UI for this trigger.',
+        $ref: BasicDisplaySchema.id
+      },
+      operation: {
+        description: 'Powers the functionality for this trigger.',
+        $ref: BasicActionOperationSchema.id
```

Comment:
Are we sure we want to reuse `BasicActionOperationSchema` for `ReadSchema.operation`? `BasicActionOperationSchema` has `perform`, `performGet` and `performResume`. Is a bulk read going to support all that? If not, we probably should come up with a new `Basic(Bulk)ReadOperationSchema'.

Reply:
Let me look at it a bit more tmrw AM and get back, but sounds like that might be the way to go here.

Reply:
I believe I have this working in dev with `BasicActionOperationSchema`... Since @xavdid approved this already, I'm thinking I want to merge it, work with it in core for my first app (Salesforce), and revise. I'm assuming while getting through the SF integration I'll have to circle back here and make more adjustments as I go too